### DISCUSSION TITLE
How to forward set-cookie headers from action

### DISCUSSION THREAD

**Discussion by gradisarjoze** [hasura/graphql-engine#7684] [Question]
We are attempting to pass cookies from an action to the fronted via Hasura action and can't seem to figure it out.

Setup:
Frontend app is hosted on _domain.com_, Hasura instance has setup a custom domain at _api.domain.com_. Hasura action is pointing to a cloud function hosted on GCP. CORS is set up to allow _domain.com_. But I'm guessing, since Hasura is in the middle, there must be some missing configuration.

A similar issue: #1654

Is there any way to pass response cookies from the cloud function to frontend via Hasura?

Any direction will be highly appreciated. Thanks!

**Comment by mkuchak** [comment]
I was having the same problem, and apparently Hasura only sends cookies via actions if it is a mutation and not a query.
